Corona-Warn-App/src/main/res/values/strings.xml

```diff
@@ -1253,6 +1253,8 @@
     <string name="test_api_calculate_risk_level">"Calculate Risk Level"</string>
     <!-- NOTR -->
     <string name="test_api_switch_background_notifications">"Background Notifications"</string>
+    <!-- NOTR -->
+    <string name="crash_report_title">Error #%1$d</string>
```

Comment:
Until we have the green light to include this in production, let's not add any strings, debug/test related UIs can be hardcoded. Before this can go into prod, we'll need approval for all strings anyways.

Reply:
thats true.
Adjusted.